robotbase/src/main/java/frc/robot/commands/intake/IntakeNoteCommand.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+/*
+Command needs to:
```

Comment:
Good self-documented code does not need comments unless it is explaining why the code is written a certain way.

Reply:
oh sorry, I left that as a note to myself and forgot to delete it